docs/management/additional-packages.md

```diff
@@ -65,28 +64,6 @@ The controller domain is not an all-purpose Linux system. It must remain minimal
 Additional packages are not meant to be in the base installation. They are only present for convenience. Unless considered truly critical, the security updates on these packages is best effort.
 :::
 
-#### Libreswan
```

Comment:
@bleader I removed this as 8.2 is EOL, but if it happens to be true for 8.2 too, I'll bring it back.

Reply:
:+1: 2 of the CVEs are not impacting 4.12, and the last one has a the fix backported in 8.3, so all good.